packages/react-lowcode/src/codegen/generation/react-components/react-intl/intl-formatter.ts

```diff
@@ -89,7 +89,7 @@ export default class ReactIntlFormatter {
   }
 
   localizePropertyNameUsingTag(prop: Property, entity?: Entity): ts.JsxSelfClosingElement {
-    let messageId: ts.StringLiteral = factory.createStringLiteral((entity ? entity.getName() : '') + "." + prop.getName())
+    let messageId: ts.StringLiteral = factory.createStringLiteral((entity ? entity.getName().toLowerCase() : '') + "." + prop.getName())
```

Comment:
what about having helper function that normalizes localization messageID for both tag and imperative call

Reply:
it should be function with two parameters: `intlMessageIdFromEntityField(entityName, fieldName)`